app/routes/intro.js

```diff
@@ -50,117 +50,20 @@ export default Ember.Route.extend({
           function: null,
           separator: " "
         }
-      },
-      exampleRows:[{
-        lon: null,
-        lat: null,
-        number: null,
-        street: null,
-        unit: null,
-        city: null,
-        district: null,
-        region: null,
-        postcode: null
-      },
-      {
-        lon: null,
-        lat: null,
-        number: null,
-        street: null,
-        unit: null,
-        city: null,
-        district: null,
-        region: null,
-        postcode: null
-      },
-      {
-        lon: null,
-        lat: null,
-        number: null,
-        street: null,
-        unit: null,
-        city: null,
-        district: null,
-        region: null,
-        postcode: null
-      },
-      {
-        lon: null,
-        lat: null,
-        number: null,
-        street: null,
-        unit: null,
-        city: null,
-        district: null,
-        region: null,
-        postcode: null
-      },{
-        lon: null,
-        lat: null,
-        number: null,
-        street: null,
-        unit: null,
-        city: null,
-        district: null,
-        region: null,
-        postcode: null
-      },
-      {
-        lon: null,
-        lat: null,
-        number: null,
-        street: null,
-        unit: null,
-        city: null,
-        district: null,
-        region: null,
-        postcode: null
-      },
-      {
-        lon: null,
-        lat: null,
-        number: null,
-        street: null,
-        unit: null,
-        city: null,
-        district: null,
-        region: null,
-        postcode: null
-      },
-      {
-        lon: null,
-        lat: null,
-        number: null,
-        street: null,
-        unit: null,
-        city: null,
-        district: null,
-        region: null,
-        postcode: null
-      },
-      {
-        lon: null,
-        lat: null,
-        number: null,
-        street: null,
-        unit: null,
-        city: null,
-        district: null,
-        region: null,
-        postcode: null
-      },
-      {
-        lon: null,
-        lat: null,
-        number: null,
-        street: null,
-        unit: null,
-        city: null,
-        district: null,
-        region: null,
-        postcode: null
-      }],
+      }
     });
+    var exampleRows = Array(10).fill({
```

Comment:
Hey I am sorry if I confused you with my previous comment. Array.fill with object will deep copy the object. so even if you want to tweak one object in array, it will change all the objects.

Reply:
@hanbyul-here I changed it to use `.map` as you suggested! In the future we can set the number (currently 10) dynamically based on the number of rows provided by the web service response.